packages/react/src/Column/Column.test.tsx

```diff
@@ -56,6 +57,16 @@ describe('Column', () => {
     expect(component).toBeInTheDocument()
   })
 
+  columnTags.forEach((tag) => {
```

Comment:
We can delete the `'renders an additional class name'` and `'renders with a section tag'` tests before this test.